tests/Unit/Domain/Creators/Test_Rectangle.cpp

```diff
@@ -114,22 +119,6 @@ void test_rectangle() {
            {Direction<2>::lower_eta(), {0, aligned_orientation}},
            {Direction<2>::upper_eta(), {0, aligned_orientation}}}},
       std::vector<std::unordered_set<Direction<2>>>{{}});
-
```

Comment:
`Test_Interval.cpp` has a similar registration and serialization part of the test that seems like it could also be removed.